lexicon.txt

```diff
@@ -12,7 +12,7 @@ addr
 addressalignedbeats
 addrstart
 addtogroup
-advertize
+advertise
```

Comment:
The lexicon contains only misspelled or unknown words.  `advertise` is spelled correctly, so you should delete it from the lexicon.

```sh
$ echo advertise | spell
$
```

Reply:
Done, thanks.